src/frontend/pages/Profile/Profile.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./profile.css";
+import { NavLink, Outlet } from "react-router-dom";
+
+export const Profile = () => {
+  return (
+    <div>
+      <div className="flex border">
+        <NavLink to="/profile" className="link" end>
+          User Profile
+        </NavLink>
+        <NavLink to="addressDetails" className="link">
```

Comment:
/ is missing in to for address and order details

Reply:
same here we do not need to give /